CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+### Added
+- marking that the order came from Shopgate App
```

Comment:
It sounds a little bit odd. You could change it to: marking orders imported by/from Shopgate App

Reply:
done